hurumap/templates/embed/iframe.html

```diff
@@ -0,0 +1,79 @@
+{% load staticfiles %}
+{% load sass_tags %}
+<!DOCTYPE html>
+<html>
+    <head>
+        <base target="_parent" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+        <link rel="stylesheet" href="{% sass_src 'css/embed.scss' %}" id="embed-styles">
+        <script src="//cdn.jsdelivr.net/g/modernizr@2.7,respond@1.4"></script>
+        <script>
+          (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+            (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+            m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+            })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+            {% if WAZIMAP.ga_tracking_id %}
+            ga('create', '{{ WAZIMAP.ga_tracking_id }}', 'auto');
```

Comment:
I know it's all the same but since this is HURUmap rep, use `HURUMAP` e.g `HURUMAP.ga_tracking_id`